modules/apps/commerce/commerce-test/src/testIntegration/java/com/liferay/commerce/internal/object/contributor/test/CommerceReturnItemObjectEntryValuesContributorTest.java

```diff
@@ -118,7 +119,7 @@ public void setUp() throws Exception {
 
 		ObjectDefinition objectDefinition =
 			_objectDefinitionLocalService.fetchSystemObjectDefinition(
-				"CommerceReturn");
```

Comment:
Maybe better to use the `TestPropsValues.getCompanyId() method`

Reply:
Why? And should I do it in all files?

Reply:
Because the CompanyThreadLocal must be set, but we don't know where it is being set. You are including a dependency between your code and the CompanyThreadLocal. For testing purposes, the `TestPropsValues` provides a set of identifiers, for example, the company where the test is being executed, the user that is executing the test, the site where the test is being executed, etc.

You don't need the CompanyThreadLocal to be set to execute your test, that is why I think it is not a good idea to include it in the test of you have a different method that provides the same information